core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -419,7 +419,7 @@ package object config {
 
   private[spark] val SHUFFLE_FILE_BUFFER_SIZE =
     ConfigBuilder("spark.shuffle.file.buffer")
-      .doc("Size of the in-memory buffer for each shuffle file output stream. " +
+      .doc("Size (in KiB) of the in-memory buffer for each shuffle file output stream. " +
```

Comment:
Really, "in KiB unless otherwise specified"?

Same for the next property below. These two are the only two that aren't in bytes by default, and have a description already. It would be handy to add a blurb about this to all of the "MiB" default properties above this too, for consistency.